src/Analysis/GWASV2/GWASContainer.jsx

```diff
@@ -1,100 +1,105 @@
-import React, { useState, useEffect } from "react";
-import { PropTypes } from "prop-types";
-import { Space, Button, Steps, Step, Popconfirm } from "antd";
+import React, { useState } from 'react';
+import { Space, Button,  Popconfirm } from 'antd';
+import SelectStudyPopulation from './SelectStudyPopulation/SelectStudyPopulation';
+import './GWASV2.css';
 
 const GWASContainer = () => {
-    const [current, setCurrent] = useState(0);
-    const gwasSteps = [
-        {
-            title: 'Step 1',
-            description: 'Select Study Population',
-        },
-        {
-            title: 'Step 2',
-            description: 'Select Outcome Phenotypes',
-        },
-        {
-            title: 'Step 3',
-            description: 'Select Covariate Phenotype',
-        },
-        {
-            title: 'Step 4',
-            description: 'Configure GWAS',
-        }
-    ]
+  const [current, setCurrent] = useState(0);
+  const [selectedCaseCohort, setSelectedCaseCohort] = useState({});
```

Comment:
we will need a new variable for the study population. So instead of `selectedCaseCohort` make a new one `selectedStudyPopulationCohort` for example.

Reply:
This state has been updated to selectedStudyPopulationCohort and the setter to setSelectedStudyPopulationCohort per your recommendation